src/handlers/transactions.ts

```diff
@@ -0,0 +1,150 @@
+import { Contract } from '@ethersproject/contracts';
+import axios from 'axios';
+import {
+  getLocalTransactionSignatures,
+  saveLocalTransactionSignatures,
+} from './localstorage/accountLocal';
+import { web3Provider } from './web3';
+import { ZerionTransaction } from '@rainbow-me/entities';
+
+interface FourByteResult {
+  bytes_signature?: string;
+  created_at?: string;
+  hex_signature?: string;
+  id: number;
+  text_signature?: string;
+}
+
+const REGISTRY_ADDRESS = '0x44691B39d1a75dC4E0A0346CBB15E310e6ED1E86';
+const abi = [
```

Comment:
can we follow the pattern of holding abi's in separate files under `src/references` (can take a look at the other ABI's there) and exposing the ABI constant thru the references/index file